packages/components/ng-ovh-feature-flipping/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@ovh-ux/ng-ovh-feature-flipping",
+  "version": "1.0.0",
```

Delegate it to the release script: 

```suggestion
  "version": "0.0.0",
```